lambdas/shared/tests/test_preview.py

```diff
@@ -80,7 +80,7 @@ def test_fcs(self):
                 'has_warnings': True,
             },
         }
-        with TemporaryDirectory() as tmp_dir, patch('t4_lambda_shared.preview.TEMP_DIR', tmp_dir):
+        with TemporaryDirectory() as tmp_dir, patch('t4_lambda_shared.preview.TEMP_DIR', tmp_dir + os.path.sep):
```

Comment:
wow this is ugly but more correct
```suggestion
        with TemporaryDirectory() as tmp_dir, patch('t4_lambda_shared.preview.TEMP_DIR', os.path.join(tmp_dir, "", ""):
```